ios/src/TGMapViewController.h

```diff
@@ -12,6 +12,26 @@
 #import <UIKit/UIKit.h>
```

Comment:
Hmm, when I delete c159079 in the interactive rebase, I get conflicts for each commit. When I resolve each, I then diff it with tangrams master, and it removes all of the iOS code. Continuing to look into this...